src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/NuGet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0"
```

Comment:
We shouldn't be revisiting any of this right now. We're keeping the current design for every high because we know it works. Once we move to msbuild we can talk about how else we can express things.